src/rsteye/app.py

```
@@ -0,0 +1,208 @@
+        if not os.path.exists(self.image_path):
+            messagebox.showerror("Error", f"Image file '{self.image_path}' not found.")
+            self.root.destroy()
+            self._ready = False
+            return
+
+    def _schedule_next_prompt(self):
+        self._popup_active = False
+        if self.root.winfo_exists():
+            self.root.after(self.interval * 1000, self.show_image)
+
+    def show_image(self):
```

Ruff (0.15.20)

[warning] Too many statements (70 > 50)

(PLR0915)